.slurmci/test_set.jl

```diff
@@ -2,6 +2,7 @@ cpu_tests = Set(((3, "examples/DGmethods_old/ex_001_periodic_advection.jl"),
                  (3, "examples/DGmethods_old/ex_002_solid_body_rotation.jl"),
                  (3, "test/DGmethods_old/compressible_Navier_Stokes/dycoms.jl"),
                  (3, "test/DGmethods_old/compressible_Navier_Stokes/rtb_visc.jl"),
+                 (1, "test/Ocean/shallow_water/GyreDriver.jl")
```

Comment:
Can we run this with multiple MPI ranks, not needed since it's 2-D but maybe worth it (?)

Reply:
Doing in-person tests, it seemed that multiGPU was slower than single GPU due to communication. Didn't really try with multiCPU but we can increase it here if you want.

Reply:
Not surprising. Was just wondering.